src/components/SideMenu.tsx

```diff
@@ -18,7 +18,7 @@ const StyledMinimizedMenuButton = styled(IconButton)(({ theme }) => ({
 
 export const MinimizedMenuIconButton = ({ children, ...props }: IconButtonProps) => {
   return (
-    <StyledMinimizedMenuButton {...props}>
+    <StyledMinimizedMenuButton component="div" {...props}>
```

Comment:
Basert på bildet som var på saken ser det ut som problemet er at det er en button (bare for styling sin del) inni en `a`. Så hvorfor ikke flytte linken inn hit i stedet, sånn at det bare rendes i en tag? Altså at det blir bare en `<a>` (som ser ut som en knapp), i stedet for at det er en `<a>` som har en `<div>` som ser ut som en knapp som nå?

Reply:
Tror det beste er om vi har noe ala dette i `MyPagePage.tsx` feks:
```
<SideMenu
  expanded={expandMenu}
  minimizedMenu={
    <IconButton
      title={t('my_page.my_page')}
      component={Link}
      to={{ pathname: UrlPathTemplate.MyPageMyMessages, search: locationState?.previousSearch }}
      onClick={() => ticketsQuery.refetch()}>
      <FavoriteBorderIcon />
    </IconButton>
  }>
```

Men den klager på `to` om jeg bruker `<MinimizedMenuIconButton>` i stedet for `IconButton`, så enten gjør vi noe rart med typen der, eller så må vi gjøre noe triksing for at det skal virke 🤔